src/core/APIWrapper.ts

```diff
@@ -57,7 +59,7 @@ const makePostString: <T_PostJSON extends {[K: string]: string}>(forString: T_Po
 	type: "urlEncoded",
 	urlEncodedData: PostURLEncoded(forString)
 })
-const makePostJSON: <T_PostJSON extends object>(jsonData: T_PostJSON) => PostJSON<T_PostJSON> = jsonData => ({type: "json", jsonData})
+export const makePostJSON: <T_PostJSON extends object>(jsonData: T_PostJSON) => PostJSON<T_PostJSON> = jsonData => ({type: "json", jsonData})
```

Comment:
This used to be needed but I replaced this with sendJson (and sendFormUrlEncoded) so you should no longer need to wrap ever post object in this